termux-exec.c

```diff
@@ -231,8 +236,33 @@ int execve(const char* filename, char* const* argv, char* const* envp)
 		}
 	}
 
-	int ret = real_execve(filename, argv, envp);
+	int ret = real_exec_fn(filename, argv, envp);
 	free(new_argv);
 	free(new_envp);
 	return ret;
+	
+}
+
+int _execvp(const char* filename, char* const* argv, char* const* envp)
+{
+        (void)envp;
```

Comment:
It looks wrong to discard the `envp` argument. This might change the target program's behavior.

Reply:
According to the function definition, execvp() does not take environment variables.